docs/usage.md

```diff
@@ -194,6 +196,23 @@ Please note the following requirements:
 2. The metadata file has to follow the [QIIME2 specifications](https://docs.qiime2.org/2019.10/tutorials/metadata/)
 3. In case of multiple sequencing runs, specific naming of samples are required, see [here](#--multipleSequencingRuns)
 
+The first column in the metadata file is the identifier (ID) column and defines the sample or feature IDs associated with your study.
+Additional columns defining metadata associated with each sample or feature ID are optional.
+Metadata files are not required to have additional metadata columns, so a file containing only an ID column is a valid QIIME 2 metadata file.
```

Comment:
This might be true, but without additional columns its useless because there are no groupings for additional analysis. But there might be more output from ampliseq, actually I am not sure since I never tested this.